--- /opt/vagrant/embedded/gems/2.1.1/gems/vagrant-2.1.1/plugins/providers/virtualbox/action/network_fix_ipv6.rb.orig	2016-12-07 23:51:00.000000000 +0300
+++ /opt/vagrant/embedded/gems/2.1.1/gems/vagrant-2.1.1/plugins/providers/virtualbox/action/network_fix_ipv6.rb	2017-02-03 04:25:00.000000000 +0300
@@ -31,10 +31,10 @@ module VagrantPlugins
             options = scoped_hash_override(options, :virtualbox)
             next if options[:ip].to_s.strip == ""
 
-            if IPAddr.new(options[:ip]).ipv6?
-              has_v6 = true
-              break
-            end
+#            if IPAddr.new(options[:ip]).ipv6?
+#              has_v6 = true
+#              break
+#            end
           end
 
           # Call up
